website/docs/developer/research/research-localstack.md

```diff
@@ -59,6 +59,323 @@ Docker's localhost can be accessed on the parent machine at http://host.docker.i
     ```
     [Full example here.](https://git.cr.usgs.gov/LPDA/appeears/-/blob/feature/rds-setup/deployment/terraform/dev/main.tf)
 
+
+### Setup details
+
+-	Make sure you have Docker desktop installed in your laptop and approved by EROS helpdesk.
+- There are two ways to install localstack- either directly using docker command or by running a docker-compose.yaml file. The recommended way is by using docker compose since all changes will stay in one config file that can be changed according to use cases.
+#### Docker compose
+
+A sample docker-compose.yml file is shown below: 
+
+        version: "3.8"
+
+        services:
+          localstack:
+            container_name: localstack-container
+            image: localstack/localstack
+            ports:
+              - "53:53"
+              - "443:443"
+              - "4566:4566"
+              - "4571:4571"
+            environment:
+              - DEBUG=1
+              - DATA_DIR=/tmp/localstack/data # Local directory for saving persistent data
+              - DEFAULT_REGION=us-east-1
+              - LAMBDA_EXECUTOR=local #running lambda from local dir
+              - LAMBDA_REMOTE_DOCKER=0
+              - DOCKER_HOST=unix:///var/run/docker.sock
+              - HOST_TMP_FOLDER=/tmp/localstack
+            volumes:
+              - "/var/run/docker.sock:/var/run/docker.sock"
+
+From your windows cmd, run the following command to start localstack: `docker compose up`
+
+        C:\Users\rhassan\localstack-test>docker compose up
```

Comment:
I would wrap this in a code block with `commandline` type. You may want to change your path to be more generic `C:\Users\rhassan\localstack-test>` by removing your username.